src/Calypso-SystemQueries/ClyScopedSystemEnvironment.class.st

```diff
@@ -90,7 +101,12 @@ ClyScopedSystemEnvironment >> extractPackageNameFrom: aDefinitionString [
 	^ (CDFluidClassDefinitionParser parse: aDefinitionString) packageName
 ]
 
-{ #category : 'accessing' }
+{ #category : 'comparing' }
+ClyScopedSystemEnvironment >> identityHash [
+	^ self scope label identityHash
```

Comment:
@tesonep I implemented this to be able to use these system environments as keys in the dictionary `defaultGlobalEnvironments`. It doesn't smell good, but I couldn't think of another way, could you please give me your feedback on it?

Reply:
It is strange that the GlobalEnvironment is an identityDictionary. 
I will put a comment and mark it as something we need to fix